gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,6 +66,13 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def raise_failover_event
+      require "awesome_spawn"
+      AwesomeSpawn.run("rake evm:raise_server_event",
+                       :chdir  => RAILS_ROOT,
```

Comment:
it is defined in ```util/postgres_admin``` which is required for ```failover_monitor```.
Is it make sense to define another constant or it is OK to use one from ```postgress_admin``` ?